test/parallel/test-namespaced-http2.js

```diff
@@ -0,0 +1,7 @@
+'use strict';
+
+const assert = require('assert');
+const oldHTTP2 = require('http2');
+const newHTTP2 = require('@nodejs/http2');
+
+assert.equal(oldHTTP2, newHTTP2, 'modules are the same');
```

Comment:
This in particular seems like a really important thing to figure out. Should the namespaced and non namespaced versions of the API return the same singleton.

I think yes.

Reply:
I think absolutely yes, or else things could break.

Reply:
As a side-note, it was briefly brought up by @jasnell in https://github.com/nodejs/TSC/issues/389#issuecomment-392528653